windows-apps-src/design/style/reveal.md

```diff
@@ -134,15 +134,23 @@ These effects are defined by two brushes:
 In most cases we handle the usage of both of them by turning Reveal on automatically for a certain controls. However, other controls will need to be enabled through applying a style, or changing their templates directly.
 
 ### When to add Reveal
-You can add Reveal to your custom controls--but before you do, consider the type of control and how it behaves. 
-* If your custom control is a single interactive element and doesn't have similar controls sharing it's space (such as menu items in a menu), it's likely that your custom control doesn't need Reveal.  
-* If you have a grouping of related interactive content or elements, then it's likely that that region of your app does need Reveal - this is commonly referred to as a [Commanding](https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/collection-commanding) surface.
+You can add Reveal to your custom controls--but before you do, consider the type of control and how it behaves.
 
-For example, a button by itself shouldn't use reveal, but a set of buttons in a command bar should use Reveal.
+Add Reveal when
+* **Your control is in a group of other similarly-sized, interactive items**
+Enable Reveal on all of the controls in the group. The Reveal effect will make it easier for users to see which specific control they're interacting with.
+* **Your custom control part of a [Commanding Collection](https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/collection-commanding)?**
+Reveal is a great choice for situations like these.
+
+Don't add Reveal when
+* **Your control isn't near hero content or is far away from other interactive elements**
+Reveal is not the best choice for these scenarios.
+
+In general, Reveal highlight brings attention to certain areas of your app, or to pieces of interactive UI that you want most of your user's focus to be drawn to. Adding Reveal to secondary UI can cause distraction, or mislead their attention; however, adding it to primary UI or hero moments in your app is where Reveal really shines.
```

Comment:
>In general, Reveal highlight brings attention to certain areas of your app, or to pieces of interactive UI that you want most of your user's focus to be drawn to.

I don't agree with this statement. It makes it sound like it is a way to highlight the UI. It is not when the UI is in rest state. It only highlights when hovered over. So this statement is inaccurate.

Reply:
> It is not when the UI is in rest state. It only highlights when hovered over. So this statement is inaccurate.

This is inaccurate. There is border approach which applies to controls when they are in rest state and when hovered. Therefore when you mouse around an area and you see the edges of things you are not directly hovered over, you will be drawn to them visually. This is what I meant by "UI that you want most of your user focus to be drawn to".

Reply:
But the way you write makes me interpret as if you are getting attention without me moving mouse.

How about a variation of something like
"In general, Reveal highlight brings user's focus to interactive elements while user is moving mouse to get to them. This brings additional attention to certain actionable items."

Reply:
Thanks! I actually just pushed some changes that also involved rewording this section. Please see those, I think I incorporated a but more of what you've mentioned in the above sentence. If you still don't like it I can use a variation on what you've written.

Reply:
Sorry but that still doesn't communicate the fact that reveal is only useful when user is moving mouse.

Reply:
This one is still something I want you to consider. I am resolving most of the rest as I saw the latest.

Reply:
Okay cool! That's fine, just want to make sure you were giving feedback on the latest :)